CODEOWNERS

```diff
@@ -183,7 +183,7 @@
 /bundles/org.openhab.binding.sonyprojector/ @lolodomo
 /bundles/org.openhab.binding.spotify/ @Hilbrand
 /bundles/org.openhab.binding.squeezebox/ @digitaldan @mhilbush
-/bundles/org.openhab.binding.modbus.sunspec/ @mrbig
+/bundles/org.openhab.binding.surepetcare/ @renescherer @HerzScheisse
```

Comment:
You should not replace other binding entries

Reply:
this has been fixed